src/languageservice/serviceclient.ts

```diff
@@ -314,11 +314,11 @@ export default class SqlToolsServiceClient {
     private createResourceClient(resourcePath: string): LanguageClient {
         // Options to control the language client
         let clientOptions: LanguageClientOptions = {
-            documentSelector: ['sql'],
-            synchronize: {
-                configurationSection: 'mssql'
-            },
-            errorHandler: new LanguageClientErrorHandler(this._vscodeWrapper)
+            // documentSelector: ['sql'],
```

Comment:
Suggest just removing this - and possibly leaving a comment explaining why you aren't setting any options.

Reply:
Not sure how it showed up again, I removed this...